linguatec_lexicon/templates/linguatec_lexicon/datavalidator.html

```diff
@@ -18,6 +18,33 @@ <h1>{{ title }}</h1>
             <form action="" enctype="multipart/form-data" method="post">
                 {% csrf_token %}
                 {{ form.as_p }}
+                <div class="input-group rg-search">
+                    <label for="lexicon_code">Código lexicon</label>
+                    <select id="selected_lex" name="lexicon_code">
+                        {% for lexicon in lexicon_list %}
+                            <optgroup label="{{ lexicon.src_language }}"> 
+                            {% for lex in lexicon_list %}
+                                {% if lex.src_language == lexicon.src_language %}
+                                    <option value="{{ lex.code }}">{{ lex.code }}</option> 
+                                {% endif %}
+                            {% endfor %}
+                        {% empty %}
+                        <option selected value=''>Empty Value</option>
+                        {% endfor %}
```

Comment:
These two nested loops are traversing the same list. Is that correct?
If it is correct, may we remove the nested loop, writing directly <option value="{{ lexicon.code }}">{{ lexicon.code }}</option>

Reply:
Yeah my bad, I wrote it a bit fast, the idea is to use the src_languages (appliyng a distinct) as labels to order the lexicon in the display list. I just have rewritten and fixed it, thank you for pointing it out.